.gitignore

```diff
@@ -2,3 +2,4 @@ out
 node_modules
 download
 .vscode
+checker-framework-language-server-0.2.0.vsix
```

Comment:
Use a wildcard to not make this version specific. Probably also better in a separate PR.

Reply:
Note the red icon at the end of the file. If you open a PR make sure you don't see that symbol and add the required newlines at the ends of files. Maybe there is an editor config that you can change.